easybuild/easyconfigs/m/mimalloc/mimalloc-1.7.2-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'mimalloc'
+version = '1.7.2'
+
+homepage = 'https://microsoft.github.io/mimalloc/'
+description = """mimalloc is a general purpose allocator with excellent performance characteristics."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+github_account = 'microsoft'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['b1912e354565a4b698410f7583c0f83934a6dbb3ade54ab7ddcb1569320936bd']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake', '3.20.1'),
+]
+
+sanity_check_paths = {
+    'files': ['lib/mimalloc-%(version_major_minor)s/libmimalloc.a', 'lib/libmimalloc.%s' % SHLIB_EXT],
+    'dirs': ['include/mimalloc-%(version_major_minor)s'],
+}
+
+runtest = 'test'
+
+# jemalloc can be used via $LD_PRELOAD, but we don't enable this by
+# default, you need to opt-in to it
```

Comment:
I was very puzzled by the mention to `jemalloc` here, I would use the same text found in mimalloc description
```suggestion
# mimalloc is a drop-in replacement for malloc and can be used in other programs without code changes,
# but we don't enable this by default, you need to opt-in by setting $LD_PRELOAD as follows
```

Reply:
copy pasted from jemalloc and i forgot to replace the name